packages/styles/stories/Components/calendar/legend.example.html

```diff
@@ -0,0 +1,275 @@
+                    </div>
+                </td>
+                <td role="gridcell" class="fd-calendar__item fd-calendar__item--legend-3">
+                    <div class="fd-calendar__text-wrapper">
+                        <span class="fd-calendar__text" role="button">24</span>
+                    </div>
+                </td>
+                <td role="gridcell" class="fd-calendar__item fd-calendar__item--weekend fd-calendar__item--legend-9">
+                    <div class="fd-calendar__text-wrapper">
+                        <span class="fd-calendar__text" role="button">25</span>
+                    </div>
+                </td>
+                <td role="gridcell" class="fd-calendar__item fd-calendar__item--weekend">
+                    <div class="fd-calendar__text-wrapper">
+                        <span class="fd-calendar__text" role="button">26</span>
+                    </div>
+                </td>
+            </tr>
+            <tr class="fd-calendar__row">
+                <td role="gridcell" class="fd-calendar__item">
+                    <div class="fd-calendar__text-wrapper">
+                        <span class="fd-calendar__text" role="button">27</span>
+                    </div>
+                </td>
+                <td role="gridcell" class="fd-calendar__item">
+                    <div class="fd-calendar__text-wrapper">
+                        <span class="fd-calendar__text" role="button">28</span>
+                    </div>
+                </td>
+                <td role="gridcell" class="fd-calendar__item fd-calendar__item--other">
+                    <div class="fd-calendar__text-wrapper">
+                        <span class="fd-calendar__text" role="button">1</span>
+                    </div>
+                </td>
+                <td role="gridcell" class="fd-calendar__item fd-calendar__item--other">
+                    <div class="fd-calendar__text-wrapper">
+                        <span class="fd-calendar__text" role="button">2</span>
+                    </div>
+                </td>
+                <td role="gridcell" class="fd-calendar__item fd-calendar__item--other">
+                    <div class="fd-calendar__text-wrapper">
+                        <span class="fd-calendar__text" role="button">3</span>
+                    </div>
+                </td>
+                <td role="gridcell" class="fd-calendar__item fd-calendar__item--other">
+                    <div class="fd-calendar__text-wrapper">
+                        <span class="fd-calendar__text" role="button">4</span>
+                    </div>
+                </td>
+                <td role="gridcell" class="fd-calendar__item fd-calendar__item--other">
+                    <div class="fd-calendar__text-wrapper">
+                        <span class="fd-calendar__text" role="button">5</span>
+                    </div>
+                </td>
+            </tr>
+            <tr class="fd-calendar__row--legend">
```

Comment:
<img width="703" alt="Screenshot 2024-10-21 at 2 12 59 PM" src="https://github.com/user-attachments/assets/4eb15763-bab5-4724-9d7f-95c5f58cbcac">

Markup wise this shouldn't be a part of the Table row. The legend should be a separate unit, that can be placed on the bottom or on the right/left side of the actual calendar. 
I think the best way to construct is something like this:
```
<div class="fd-calendar__legend" role="list">
    <div class="fd-calendar__legend-item" role="listitem">
        <span class="fd-calendar__legend-marker"></span>
        <span class="fd-calendar__legend-label">Square marker</span>
    </div>
    <div class="fd-calendar__legend-item" role="listitem">
        <span class="fd-calendar__legend-marker fd-calendar__legend-marker--circle"></span>
        <span class="fd-calendar__legend-label">Circle marker</span>
    </div>
    <div class="fd-calendar__legend-item" role="listitem">
        <span class="fd-calendar__legend-marker fd-calendar__legend-marker--background"></span>
        <span class="fd-calendar__legend-label">With background marker</span>
    </div>
</div>
```

Reply:
`<div class="fd-calendar__legend" role="list">` should be a sibling element to the table element with class `fd-calendar__content`. This would make the legend "independent" and can be on the bottom or on the side of the table.